src/test/java/core/basesyntax/service/FileWriterTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.impl.FileWriterImpl;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterTest {
+    private static final String dayReportTestFilePath = "src/main/resources/dayreport.csv";
+    private static final String FILE_HEADER = "fruit,quantity";
+    private static final long FILE_REPORT_SIZE = 46;
+    private static List<String> testList = new ArrayList<>();
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        testList.add(FILE_HEADER);
+        testList.add(System.lineSeparator());
+        testList.add("banana,30");
+        testList.add(System.lineSeparator());
+        testList.add("pineapple,22");
+        testList.add(System.lineSeparator());
+        testList.add("apple,50");
+        File file = new File(dayReportTestFilePath);
+        file.delete();
+    }
+
+    @Test
+    public void writeToFile_createReportFile_OK() {
+        FileWriter fileWriter = new FileWriterImpl();
+        fileWriter.writeToFile(dayReportTestFilePath, testList);
+        File reportFile = new File(dayReportTestFilePath);
+        assertTrue(reportFile.exists());
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void writeToFile_nullList_NotOK() {
+        List<String> testList2 = null;
```

Comment:
It's bad practice to use numbers in variable names

Reply:
OK